<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deytoyslinux</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>181</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(47, 47, 47);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="fsrmatters_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Farmatters</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convertors_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Converters</string>
     </property>
    </widget>
    <widget class="QPushButton" name="all_tools_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>All Tools</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>861</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 44, 44);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home_page">
     <widget class="QFrame" name="home_frame">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>861</width>
        <height>651</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="jsonformate_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>JSON Farmatter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="jsontoyaml_button">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>JSON TO YAML</string>
       </property>
      </widget>
      <widget class="QPushButton" name="yamltojson_button">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>40</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string> YAML TO JSON</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="farmatters_page">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>861</width>
        <height>651</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="jsonformate_button_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>JSON Farmatter</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="converters_page">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>861</width>
        <height>651</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="jsontoyaml_button_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>JSON TO YAML</string>
       </property>
      </widget>
      <widget class="QPushButton" name="yamltojson_button_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string> YAML TO JSON</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="json_formate_page">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>861</width>
        <height>651</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="json_input">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>771</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="json_output">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>330</y>
         <width>771</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>300</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
      <widget class="QPushButton" name="json_format_button">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>296</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="jsontoyaml_page">
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>861</width>
        <height>651</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="jsontoyaml_convert_button">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>286</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="yaml_output_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>771</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="json_input_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>771</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="yamltojson_page">
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>861</width>
        <height>651</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="yamltojson_convert_button">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>296</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="yaml_input_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>771</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="json_output_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>771</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
